app/models/branding_group.rb

```diff
@@ -60,6 +60,23 @@ def facets
     self.facet_list.any? ? SearchFacet.where(:identifier.in => self.facet_list) : SearchFacet.visible
   end
 
+  # list of curator emails
+  def curator_list
+    users.map(&:email)
+  end
+
+  # determine if user can edit branding group (all portal admins & collection curators)
+  def can_edit?(user)
+    return false if user.nil?
+
+    !!(user.admin? || users.include?(user))
```

Comment:
love it

Reply:
This actually should be `!!(user && (user.admin? || users.include?(user)))`, because otherwise if the user is `nil` it returns `nil`.  While this is still "falsey", I prefer to constrain the response to `true` or `false`.

Reply:
e6441de